Automattic/jetpack
# Update the IA and navigation/tabs

Addresses #4116

![ia-demo](https://cloud.githubusercontent.com/assets/7129409/16602657/de3d2ef2-42e3-11e6-8463-5a520b90b18f.gif)

**Notable Notes:**
- No more `Health` tab/route
- No more `More` tab/route
- New route/tab `Appearance`
- New route/tab `Writing`
- Manage is no longer shown if active 
- Omnisearch has been removed from the UI and nav menu (the code remains)
- Stats removed from subnav menu

**To test:** 
- Make sure that all routes look good when navigating. 
- Make sure that the subnav menu is highlighted appropriately when you're on the respective "page" 
- Make sure all the module toggles and cards work as they did before
- Make sure you don't see `manage` card if it's active
- Make sure that both omnisearch and stats pages are navigable, even though they're not in the sub-nav menu anymore. 
`/wp-admin/index.php?page=omnisearch`
`/wp-admin/index.php?page=stats`
